crates/forge/tests/e2e/fork.rs

```diff
@@ -0,0 +1,26 @@
+use indoc::indoc;
+
+use crate::e2e::common::runner::{runner, setup_package};
+
+#[test]
+fn print_error_if_attributes_incorrect() {
+    let temp = setup_package("fork");
+    let snapbox = runner();
+
+    snapbox
+        .current_dir(&temp)
+        .arg("fork::")
+        .assert()
+        .stderr_matches(indoc!
+        {r#"error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
+        --> lib.cairo:2:7
+       #[fork(url: "https://test.com")]
+             ^***********************^
+       
+       error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
+        --> lib.cairo:2:7
+       #[fork(url: "https://test.com")]
+             ^***********************^
+       
```

Comment:
Can we add an issue to investigate what causes diagnostic errors to appear twice? It is a general issue, not related to this PR

Reply:
#699